site/docs/pages/swap/swap-settings.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: <SwapSettings /> · OnchainKit
+description: Swap component slippage controls
+---
+
+import { Avatar, Name } from '@coinbase/onchainkit/identity';
+import {
+  Swap,
+  SwapAmountInput,
+  SwapButton,
+  SwapMessage,
+  SwapSettings,
+  SwapSettingsSlippageDescription,
+  SwapSettingsSlippageInput,
+  SwapSettingsSlippageTitle,
+  SwapToggleButton,
+} from '@coinbase/onchainkit/swap';
+import { Wallet, ConnectWallet } from '@coinbase/onchainkit/wallet';
+import App from '../../components/App';
+import SwapWrapper from '../../components/SwapWrapper';
+import { walletDropdownLinkCustomBaseIconSvg } from '../../components/svg/walletDropdownLinkCustomBaseIconSvg';
+
+# `SwapSettings`
+The `SwapSettings` component allows users to configure the maximum acceptable price difference (slippage) for their token swap transactions. 
+It provides a simple interface to set and adjust this tolerance level.
+
+## Usage 
```

Comment:
Let's rewrite all of this as Steps and story telling, like we have here https://onchainkit.xyz/transaction/transaction

Reply:
Synced with Tina - Going to merge this"v1" now to focus on trying to get USD values in before the release tomorrow. 

After USD values are implemented I will update `SwapSettings` to the story telling design that we have for `/transaction`

Reply:
Update - merging this PR after Thursday's release. There is a icon bug fix that we need